### DISCUSSION TITLE
Menus undefined

### DISCUSSION THREAD

**Discussion by AeShevch** [WebDevStudios/nextjs-wordpress-starter#798] [Q&A]
Greetings! Thank you for a great tool!

Please help me figure it out - I'm transferring the site from a local server to a remote one.
The menu is not displayed on the remote server

On the local server:
![Снимок экрана 2021-12-21 в 0 25 51](https://user-images.githubusercontent.com/44289476/146834850-04b3d73f-d738-4588-91e4-f81c0a33349b.png)

On a remote server:
menus: undefined

Wordpress on a remote server is a complete copy of the local one - copied through the Duplicator plugin

I replaced the values of WORDPRESS_URL and NEXT AUTO_URL in .env with the URL of the remote server.

**Comment by itsamoreh** [comment]
@AeShevch sorry for the late reply! Is this still an issue? Have you double checked that the menus exist in the WordPress admin?

**Reply by AeShevch** [reply]
@itsamoreh Thanks for the answer anyway! But I solved the problem, it was the wrong nginx settings. 

Now I have another question – how to make new pages created in the admin panel open on the site? Now they all, except the main one, return 404.

**Reply by itsamoreh** [reply]
@AeShevch are you on the latest `canary` version of the starter? Pages should just work by default. Have you tried restarting the dev server after you add a new page?
